### ISSUE TITLE
Unable to display GIS files generated from duckdb (v0.10.1)

### ISSUE THREAD

**Issue by pratiman-91** [duckdb/duckdb_spatial#288]
Unable to display GIS (QGIS 3.34.3-Prizren) file generated by v0.10.1. 

```sql
INSTALL httpfs;
INSTALL spatial;

LOAD httpfs;
LOAD spatial;
SET s3_region='us-west-2';

COPY (
    SELECT
        height,
        num_floors,
        ST_GeomFromWKB(geometry) as geometry
    FROM
        read_parquet('s3://overturemaps-us-west-2/release/2024-03-12-alpha.0/theme=buildings/type=*/*', hive_partitioning=1)
    LIMIT
        100
    ) TO 'buildings_sample_0_10_1.fgb'
WITH (FORMAT GDAL, DRIVER 'FlatGeobuf');
```
Actual Output:
![image](https://github.com/duckdb/duckdb_spatial/assets/31694629/fe3e4174-7488-48a8-b47c-690ad1bb50e0)

Expected Output:
from v0.10.0

![image](https://github.com/duckdb/duckdb_spatial/assets/31694629/a90db033-8c17-4ca6-8f22-a1e1fed10572)

**Comment by Maxxen** [comment]
Thanks again for reporting this, I have a fix ready in #296

**Comment by pratiman-91** [comment]
Thanks for the fix. It has resolved the issue.
